content/concepts/transports/webrtc.md

```diff
@@ -1,9 +1,122 @@
 ---
 title: "WebRTC"
-description: "WebRTC is a new specification that uses QUIC to offer an alternative to WebSocket. Conceptually, it can be considered WebSocket over QUIC.Learn about WebTransport and how it is used in libp2p."
+description: "WebRTC is a protocol for real-time communication and is used to establish connections between browsers and other nodes."
 weight: 110
 aliases:
     - "/concepts/transports/webrtc"
 ---
 
-Coming soon!
+## What is WebRTC?
+
+[WebRTC (Web Real-Time Communications)](https://webrtc.org/) is a framework for real-time
+communication and is used to establish browser-to-server and browser-to-browser connections
+between applications.
+
+WebRTC was originally designed to make audio, video, and data
+communication between browsers user-friendly and easy to implement.
+It was first developed by [Global IP Solutions (or GIPS)](http://www.gipscorp.com/).
+In 2011, GIPS was acquired by Google where the [W3C](https://www.w3.org/) started to work
+on a standard for WebRTC.
+
+It serves as a good choice for applications that need built-in support
+for media communication and do not have specific requirements for the underlying
+transport protocol.
+
+## WebRTC in libp2p
+
+In libp2p, WebRTC is used to connect from browsers to other nodes. libp2p WebRTC only uses
+WebRTC data channels and does not make use of any of the multimedia features.
+
+### Key features
+
+- Peer connections: WebRTC enables
+  [direct peer-to-peer connections](https://webrtc.org/getting-started/peer-connections)
+  between browsers and other nodes.
+
+- Data channels: WebRTC provides peer-to-peer data channels called
+  [WebRTC data channels](https://developer.mozilla.org/en-US/docs/Games/Techniques/WebRTC_data_channels),
+  which works on
+  [SCTP (Stream Control Transmission Protocol)](https://en.wikipedia.org/wiki/Stream_Control_Transmission_Protocol).
+  A WebRTC data channel allows applications to send a text or binary data over an active connection to a peer.
+
+- [NAT traversal](../nat/overview): WebRTC includes mechanisms (like
+  [ICE](https://datatracker.ietf.org/doc/rfc5245/)) to connect to nodes that run behind
+  NATs and firewalls. In non-decentralized WebRTC, this can be facilitated by a
+  [TURN server.](https://webrtc.org/getting-started/turn-server),
+  but other signaling channels, such as WebSocket running on a central server, can also be used.
```

Comment:
Should we describe what a signaling protocol is vs a transport protocol in here or in the glossary? i.e. something like https://www.onsip.com/voip-resources/voip-fundamentals/webrtc-signaling

Reply:
I'd suggest somewhere we can link out to

Reply:
Can we link to that page? It's not ideal to introduce a concept without explaining it (esp if its on the 1st introduction.)